resources/views/errors/layout.blade.php

```diff
@@ -1,70 +1,32 @@
-<!DOCTYPE html>
-<html lang="ja">
-    <head>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-
-        <title>@yield('title')</title>
-
-        <!-- Styles -->
-        <style>
-            html, body {
-                background-color: #fff;
-                color: #304554;
-                font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-                height: 100vh;
-                margin: 0;
-            }
-
-            .full-height {
-                height: 100vh;
-            }
-
-            .flex-center {
-                align-items: center;
-                display: flex;
-                justify-content: center;
-            }
-
-            .position-ref {
-                position: relative;
-            }
-
-            .content {
-                text-align: center;
-            }
-
-            .app-name {
-
-            }
-
-            .app-name a {
-                color: #304554;
-                text-decoration: none;
-            }
-
-            .app-name a:hover {
-                text-decoration: underline;
-            }
-
-            .title {
-                font-size: 36px;
-                font-weight: bold;
-            }
-        </style>
-    </head>
-    <body>
-        <div class="flex-center position-ref full-height">
-            <div class="content">
-                <div class="app-name">
-                    <a href="{{ url('/') }}">
-                        {{ config('app.name') }}
-                    </a>
-                </div>
-                <div class="title">
-                    @yield('message')
-                </div>
+@extends('v2.layouts.app')
+
+@section('head')
+@hasSection ('twitter')
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+@endif
+@endsection
```

Comment:
わざわざ `head` というセクションを作らなくても、 `@push('js')` するだけで対応可能です！
(no_drawer も同様)

```suggestion
@hasSection ('twitter')
@push('js')
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
@endpush
@endif
```